test/test_config.py

```diff
@@ -48,13 +50,14 @@ def test_config(self):
         cfg = MuSpinConfig(itest.evaluate())
 
         self.assertEqual(cfg.name, "muspinsim")
-        self.assertEqual(len(cfg), 8)
-        self.assertEqual(cfg.results.shape, (2, 2, 21))
+        self.assertEqual(len(cfg), 16)
+        self.assertEqual(cfg.results.shape, (2, 2, 2, 21))
 
         # Try getting one configuration snapshot
         cfg0 = cfg[0]
 
         self.assertTrue((cfg0.B == [0, 0, 1]).all())
+        self.assertTrue((cfg0.intrinsic_B == [0, 0, 1]).all())
```

Comment:
Should we also test that `self.assertIn("intrinsic_B", cfg._file_ranges)`, like we do for `"B"` on line 78? As far as I can tell they would both be treated as distinct ranges?